I see you mean this:

```
======================================================================
FAIL: test_cov (pyemma.coordinates.tests.test_tica.TestTICAExtensive)
----------------------------------------------------------------------
Traceback (most recent call last):
File "C:\projects\pyemma\pyemma\coordinates\tests\test_tica.py", line 
174, in test_cov
assert (np.max(self.tica_obj.cov - cov_ref) < 3e-2)
AssertionError
======================================================================
FAIL: test_cov_tau (pyemma.coordinates.tests.test_tica.TestTICAExtensive)
----------------------------------------------------------------------
Traceback (most recent call last):
File "C:\projects\pyemma\pyemma\coordinates\tests\test_tica.py", line 
180, in test_cov_tau
assert (np.max(self.tica_obj.cov_tau - cov_tau_ref) < 3e-2)
AssertionError
----------------------------------------------------------------------
```

This is then indeed a stochastic problem I would guess. We can set

cls.T=40000

in line 132 and additionally increase the threshold to 0.05 in lines 174 
and 180. Can you do this in your PR and retry? I don't wanna open a PR 
for this.

Thanks
Frank.

Am 16/06/15 um 16:00 schrieb Guillermo Pérez-Hernández:

> I might be wrong, but isn't that PR already taken into account in my 
> devel branch?
> https://github.com/gph82/PyEMMA/commits/devel
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/markovmodel/PyEMMA/pull/362#issuecomment-112443149.

## 

---

Prof. Dr. Frank Noe
Head of Computational Molecular Biology group
Freie Universitaet Berlin

Phone: (+49) (0)30 838 75354
Web:   research.franknoe.de

## Mail:  Arnimallee 6, 14195 Berlin, Germany

--- a/pyemma/coordinates/tests/test_tica.py
+++ b/pyemma/coordinates/tests/test_tica.py
@@ -129,7 +129,7 @@ class TestTICAExtensive(unittest.TestCase):
         # generate HMM with two Gaussians
         cls.P = np.array([[0.99, 0.01],
                           [0.01, 0.99]])
-        cls.T = 10000
+        cls.T = 40000
         means = [np.array([-1, 1]), np.array([1, -1])]
         widths = [np.array([0.3, 2]), np.array([0.3, 2])]
         # continuous trajectory
@@ -171,13 +171,13 @@ class TestTICAExtensive(unittest.TestCase):
     def test_cov(self):
         cov_ref = np.dot(self.X.T, self.X) / float(self.T)
         assert (np.all(self.tica_obj.cov.shape == cov_ref.shape))
-        assert (np.max(self.tica_obj.cov - cov_ref) < 3e-2)
+        assert (np.max(self.tica_obj.cov - cov_ref) < 5e-2)
 
     def test_cov_tau(self):
         cov_tau_ref = np.dot(self.X[self.lag:].T, self.X[:self.T - self.lag]) / float(self.T - self.lag)
         cov_tau_ref = 0.5 * (cov_tau_ref + cov_tau_ref.T)
         assert (np.all(self.tica_obj.cov_tau.shape == cov_tau_ref.shape))
-        assert (np.max(self.tica_obj.cov_tau - cov_tau_ref) < 3e-2)
+        assert (np.max(self.tica_obj.cov_tau - cov_tau_ref) < 5e-2)
 
     def test_data_producer(self):
         assert self.tica_obj.data_producer is not None

